pkg/ring/ring.go

```diff
@@ -112,6 +116,8 @@ var (
 		return s == READONLY
 	})
 
+	WriteShard = NewOp([]InstanceState{ACTIVE, PENDING, LEAVING, JOINING}, func(s InstanceState) bool { return false })
```

Comment:
Can you add some comments for this operation? The name of the operation is not very intuitive as I cannot tell the difference of it and the `Write` operation.

Reply:
Changed the way this fix works. We are not using a filter on sharding anymore. This would create a complex problem to fix on Read